INSTALL.md

```diff
@@ -22,25 +22,24 @@ Table of Contents:
 Please note: Mac users who want to use fink can skip down to Section 4.ii and
 use that to satisfy all dependencies before installing.
 
-i) Python (2.6 or 2.7 series), with some additional modules installed
----------------------------------------------------------------------
+i) Python (2.6, 2.7, or 3.4 series), with some additional modules installed
```

Comment:
I still have a system that has 2.6, so I can still test on it.  It's not a huge burden to support it.  Mostly I just need to keep adding numbers to your `{}`'s in `format` statements.  :)